docs/scripts/uploadTestFiles.py

```diff
@@ -148,83 +149,150 @@ def sync_to_synapse(path: str, project_id: str, syn: synapseclient.Synapse) -> N
 
 
 def execute_sync_to_s3(path: str, key_in_bucket: str) -> None:
-    """Executes the AWS CLI sync command. Expected to run last as this will delete local files.
-
-    :param path: The path to the root directory
-    :param test_name: The name of the test to add to the span name
-    """
-
     subprocess.run(
-        [
-            "aws",
-            "s3",
-            "sync",
-            path,
-            f"{S3_BUCKET}/{key_in_bucket}",
-            "--profile",
-            S3_PROFILE,
-        ]
+        f"aws s3 sync {path} {S3_BUCKET}/{key_in_bucket} --profile {S3_PROFILE}",
+        shell=True,
+        check=False,
     )  # nosec
 
 
-def set_up_projects_one_time(path: str, syn: synapseclient.Synapse) -> None:
-    create_folder_structure(
-        path=path,
-        depth_of_directory_tree=1,
-        num_sub_directories=5,
-        num_files_per_directory=5,
-        total_size_of_files_mbytes=1,
-    )
-    # Set up the project:
-    project_25_files_1MB = syn.store(obj=Project(name=PROJECT_25_FILES_1MB))
-    sync_to_synapse(path=path, syn=syn, project_id=project_25_files_1MB.id)
-    os.remove(f"{path}/benchmarking_manifest.tsv")
-    execute_sync_to_s3(path=path, key_in_bucket=PROJECT_25_FILES_1MB)
-    shutil.rmtree(path)
+async def set_up_folders_one_time(path: str, syn: synapseclient.Synapse) -> None:
+    project = await Project(id=PARENT_PROJECT).get_async()
 
+    depth = 1
+    sub_directories = 1
+    files_per_directory = 1000
+    size_mib = 1024
```

Comment:
isn't this supposed to be 1? `size_mib` is fed to `total_size_of_files_mib` which is used to calculate `total_size_of_file_bytes` by multiplying by `MiB` which is already 1 mib i.e. `1024 * 1024`

Reply:
Yeah, I had written this so that I passed in the total size I wanted to test the transfer with. Then the code will distribute that size evenly over the requested number of files.